src/Fields/interpolate.jl

```diff
@@ -66,14 +66,14 @@ end
 
 @inline function fractional_x_index(x, locs, grid::XRegularRG)
     x₀ = xnode(1, 1, 1, grid, locs...)
-    Δx = xspacings(grid, locs...)
+    Δx = xspacings(grid, locs...)[1, 1, 1]
```

Comment:
```suggestion
    Δx = @inbounds first(xspacings(grid, locs...))
```

But I don't really like this. It could make sense to just specialize `xspacings` for a regularly-spaced grid.

Reply:
I followed `xnode(1, 1, 1, ...)` but yeah that's a good point.

I guess we need to decide whether `xspacings` should always return a `KernelFunctionOperation` or something more useful in these cases.

My suggestion above was to add a new function or flag to return the "more useful" reduced-dimension spacings. Just not sure on naming.

Reply:
What is the underlying goal behind always returning KernelFunctionOperation when the spacing is constant?